.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!--
+Before you open this PR, make sure that you review and are taking steps to follow [the Definition of Mergeable in our team agreement](https://docs.google.com/document/d/1nwZIF_lydPWfvixxZlQLNt4nqy3Qp13pHQnMcYJjTqE/edit#heading=h.6mnhaqm79e12). You may need to scroll down to that subheader.
+-->
+
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```

Comment:
This will end up closing the ticket. Sometimes we don't want the ticket closed so that QA can approve it. Please change to (issue)